### DISCUSSION TITLE
[Bug] Searching for text on webpage on macOS opens Feedback shortcut

### DISCUSSION THREAD

**Discussion by StrangeRanger** [IntelliTect/EssentialCSharp.Web#17] [Feedback and Ideas] [answered]
On windows, you can easily search for text on the page via `Ctrl` + `F`, which poses no problem when done on [essentialcsharp.com](https://essentialcsharp.com/). On macOS, you instead use `Cmd` + `F`. Unfortunately, when doing this on macOS, instead of opening the prompt to search text within the webpage, it executes the website's "Give Feedback" option that can usually be selected via pressing the `F` key.

**Comment by Keboo** [comment]
Thank you @StrangeRanger, I have logged a bug to get this fixed.

**Reply by BenjaminMichaelis** [reply]
Thank you for the feedback @StrangeRanger!

**Comment by BenjaminMichaelis** [comment]
Looking into this issue currently @StrangeRanger, Are you having issues with `cmd` + `c` on macOS or `cmd` + `f`? I am seeing an issue with cmd + f  but not yet with cmd + c. Also; what browser(s) are you using that have this issue?

**Reply by StrangeRanger** [reply]
Oh crap. I'm sorry, I was meaning `cmd` + `f`. 

I'm using Brave, which is based on chromium.

**Reply by BenjaminMichaelis** [reply]
All good, and thanks for the clarification! I'm seeing this issue on the chromium based browsers for macOS. I will get a fix out for this!

**Reply by StrangeRanger** [reply]
Also, when I try it on firefox, it opens the find window, but I also get notified that "Firefox prevented this website from opening a pop-up window."
When allowing the pop-up, it opens the GitHub feedback page, in the same way that it does on Brave.

**Comment by BenjaminMichaelis** [comment, marked as answer]
@StrangeRanger An update has been published; please refresh your browser page to see the fix
